docs/en/core-services/Caching.md

```diff
@@ -144,12 +144,18 @@ The user can also specify the tier that the data will be written to via
 
 If the data is already in Alluxio, the client will simply read the block from where it is already stored.
 If Alluxio is configured with multiple tiers, the block may not be necessarily read from the top tier,
-since it could have been moved to a lower tier transparently.
+since it may have been loaded into a lower tier, or moved to a lower tier transparently. 
```

Comment:
@gpang Do we want to mention how the blocks could have been asynchronously loaded? Very interestingly we only mention the async caching fact in https://docs.alluxio.io/os/user/stable/en/operation/Performance-Tuning.html#async-block-caching I don't think the doc path for that feature is intuitive, but I also don't think this section is a good place for that. What do you think?